javagen/pom.xml

```diff
@@ -68,6 +68,12 @@
       <version>4.13.2</version>
       <scope>test</scope>
     </dependency>
+      <dependency>
+          <groupId>io.projectreactor.netty</groupId>
+          <artifactId>reactor-netty-http</artifactId>
+          <version>1.1.7</version>
+          <scope>compile</scope>
+      </dependency>
```

Comment:
Why is this dependency needed?

Reply:
I think IntelliJ added that when I was fixing a problem, will remove it.